components/gallery/GalleryItemAction/GalleryItemActionSlides.vue

```diff
@@ -44,6 +44,11 @@ defineProps<{
       font-size: 1rem !important;
       height: 100% !important;
     }
+
+    .o-btn {
+      font-size: 1rem;
+      height: 100%;
+    }
```

Comment:
I don't think it's a good idea to set class from here

Reply:
@roiLeo What do you think of the following idea?

Create a separate button component for the Gallery Item Action that inherits from `<NeoButton />` and overrides its font size:
```vue
// <GalleryItemActionButton />

<template>
  <NeoButton class='action-button'>...</NeoButton>
</template>

<style>
  .action-button {
    font-size: 1rem;
  }
</style>
```

Use `<GalleryItemActionButton />` instead of `<NeoButton />` in `<GalleryItemOffer />`. This way, we don't need to look elsewhere for various conditions that override the button style.

The override of the button style in the `NeoTooltip` component was indeed recently removed, which is the reason for this issue, but maybe it's not a good idea to add it back? Because it's hard to understand why the button style is overridden in a Tooltip, and it's hard to predict the impact when trying to modify the overridden style here.